package.json

```diff
@@ -42,6 +42,7 @@
         "@solana/wallet-adapter-base": "^0.9.7",
         "@solana/web3.js": "^1.47.3",
         "ethers": "^5.6.9",
+        "lucid-cardano": "^0.10.1",
```

Comment:
lets use this version https://www.npmjs.com/package/@jpg-store/lucid-cardano

its commonJS instead of web assembly, so its more environment friendly (we use it in crossmint-main as well)

Reply:
I was getting some error when i tried to use that, but ill try again